src/constants.cpp

```diff
@@ -2104,6 +2104,49 @@ const char *eyemods[] = {
   "\n"
 };
 
+const char *armsmods[] = {
+  "Obvious",
+  "Synthetic",
+  "Armor Mod1",
+  "Strength Mod1",
+  "Strength Mod2",
+  "Strength Mod3",
+  "Quickness Mod1",
+  "Quickness Mod2",
+  "Quickness Mod3",
+  "Cyberarm Gyromount"
```

Comment:
Missing terminal comma means this will be merged with the next line item-- but you also don't need the \n at the end since the code you used doesn't look for that. Your call as to whether you add a comma here, or delete the \n in the rest and remove the commas.